x_transformers/x_transformers.py

```diff
@@ -357,7 +365,12 @@ def forward(
         k = self.to_k(k_input)
         v = self.to_v(v_input)
 
+        if not collaborative_heads:
         q, k, v = map(lambda t: rearrange(t, 'b n (h d) -> b h n d', h = h), (q, k, v))
```

Comment:
missing an indentation here!

Reply:
My bad, I did not commit whitespace changes to avoid poluting the diff with removing trailing spaces in the README but removed this tab.